components/layout_2020/display_list/mod.rs

```diff
@@ -811,10 +813,10 @@ impl<'a> BuilderForBoxFragment<'a> {
             .border
             .to_physical(self.fragment.style.writing_mode);
         let widths = SideOffsets2D::new(
-            border_widths.top.px(),
-            border_widths.right.px(),
-            border_widths.bottom.px(),
-            border_widths.left.px(),
+            Length::from(border_widths.top).px(),
+            Length::from(border_widths.right).px(),
+            Length::from(border_widths.bottom).px(),
+            Length::from(border_widths.left).px(),
         );
```

Comment:
```rs
        let border_widths = self
            .fragment
            .border
            .to_physical(self.fragment.style.writing_mode)
            .to_webrender();
```

and just use `border_widths` instead of `widths`.